### PR TITLE
feat: integration test for kafka debezium sink

### PR DESCRIPTION
**Pull request by wugouzi**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?
This integration test (and demo project) contructs a pipeline as
```
RisingWave -> (Debezium json) -> Kafka -> (JDBC connector) -> MySQL/PostgreSQL
```
Please read readme to setup the pipeline. The test sql file creates a table including all the currently supported RisingWave types. The test will pass when #9755 goes into docker image. For now, you can use local RisingWave to test.

## Checklist For Contributors

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [x] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Checklist For Reviewers

- [x] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

File: integration_tests/rw-kafka-sink-db/docker-compose-local.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
```
Comment (tabVersion) [inline integration_tests/rw-kafka-sink-db/docker-compose-local.yml:1]: what is the difference between local and the other version

**Comment by tabVersion** [comment]
The test logic looks not consistent with others. 
you may refer to https://github.com/risingwavelabs/risingwave/blob/main/integration_tests/scripts/run_demos.py , the script will sequentially run `'create_source.sql', 'create_mv.sql', 'query.sql'` and then check the result.

**Comment by tabVersion** [comment]
duplicate with #9235 & #9674

File: integration_tests/rw-kafka-sink-db/docker-compose-local.yml (hunk 1 above)
Comment (wugouzi) [reply integration_tests/rw-kafka-sink-db/docker-compose-local.yml:1]: The local version requires a local running RisingWave and is for test for a newly built RisingWave that is not in official docker image yet.

**Comment by wugouzi** [comment]
> The test logic looks not consistent with others. you may refer to [`main`/integration_tests/scripts/run_demos.py](https://github.com/risingwavelabs/risingwave/blob/main/integration_tests/scripts/run_demos.py?rgh-link-date=2023-05-13T11%3A53%3A56Z) , the script will sequentially run `'create_source.sql', 'create_mv.sql', 'query.sql'` and then check the result.
> 
> may check this to init pg and MySQL in docker compose file
> 
> https://github.com/risingwavelabs/risingwave/blob/be5d30deec4be7edbb7182d012a43bfb329c1185/integration_tests/debezium-postgres/docker-compose.yml#L76-L79

This test follows the test in [`main`/integration_tests/kafka-cdc-sink/](https://github.com/risingwavelabs/risingwave/tree/main/integration_tests/kafka-cdc-sink). I will modify the `run_demos.py` to fit the tests.

File: integration_tests/rw-kafka-sink-db/docker-compose-local.yml (hunk 1 above)
Comment (tabVersion) [reply integration_tests/rw-kafka-sink-db/docker-compose-local.yml:1]: then please remove this part. it does not always work on local machine, for example, macos, local network and docker network are isolated by default.

**Comment by tabVersion** [comment]
> Why do you introduce a new test instead of modifying based on the `kafka-cdc-sink` demo?

Good catch. The thing is about testing capability between risingwave and the downstream kafka connect component. Users always try to utilize existing products instead of writing new ones.
Further, confluent is dominant in kafka hosting, we need a test to maintain the protocol capability.

**Comment by wugouzi** [comment]
> @tabVersion I didn't get it. 😂 We do have a demo called kafka-cdc-sink, developed by Renjie Liu. It uses Flink for the verification to ensure that RisingWave's output can be consumed by other systems. It might be better if you add more testing logics based on this demo.

I think there are some difference. RisingWave didn't send correct Debezium format kv pair until #9755, so some downstream connectors failed before (in this case, kafka jdbc connector). But since these two tests are similar, maybe we could combine them?

**Pull request closed without merging**